sdk/core/azure-core-amqp/src/test/java/com/azure/core/amqp/implementation/ReactorConnectionTest.java

```diff
@@ -98,7 +98,7 @@ static void afterAll() {
     @BeforeEach
```

Comment:
Do you have any tests for applicationId? I see you just passing NULL everywhere.

Reply:
Added test in ConnectionHandlerTest to check for applicstionId everywhere else it is pass through.